app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+def go_to_cafe(friends, cafe) -> str:
+    total = 0
```

Comment:
The `total` variable does not reflect what is stored in it. Maybe `total friends`,  `total in masks`, etc. use `masks_to_buy` or another more specific variable name.